.dockerignore

```diff
@@ -0,0 +1,8 @@
+.env
+.env.template
+env_vars
+.gitignore
+Vagrantfile
+.vscode
+geckodriver
+geckodriver.org
```

Comment:
I assume you meant `.log`

See above for how to remove a file that is already tracked.